gtfs-realtime/spec/en/reference.md

```diff
@@ -20,13 +20,27 @@ Version 1.0 of the feed specification is discussed and documented on this site.
             *   [TripDescriptor](#message-tripdescriptor)
                 *   [ScheduleRelationship](#enum-schedulerelationship-1)
             *   [VehicleDescriptor](#message-vehicledescriptor)
+                *   [CarriageDescriptor](#message-carriagedescriptor)
+                    *   [OccupancyStatus](#enum-occupancystatus)
+                    *   [WheelchairAccessible](#enum-wheelchairaccessible)
+                    *   [ToiletFacilities](#enum-toiletfacilities)
+                    *   [WifiAvailability](#enum-wifiavailability)
+                    *   [AirConditioning](#enum-airconditioning)
+                    *   [BicyclesAllowed](#enum-bicyclesallowed)
             *   [StopTimeUpdate](#message-stoptimeupdate)
                 *   [StopTimeEvent](#message-stoptimeevent)
                 *   [ScheduleRelationship](#enum-schedulerelationship)
         *   [VehiclePosition](#message-vehicleposition)
             *   [TripDescriptor](#message-tripdescriptor)
                 *   [ScheduleRelationship](#enum-schedulerelationship-1)
             *   [VehicleDescriptor](#message-vehicledescriptor)
+                *   [CarriageDescriptor](#message-carriagedescriptor)
+                    *   [OccupancyStatus](#enum-occupancystatus)
```

Comment:
@RachM That was my exact same reasoning for originally proposing `OccupancyStatus` in `VehicleDescriptor`.  So, I'm also interested to hear others thoughts :).